rethinkdb/datadog_checks/rethinkdb/operations.py

```diff
@@ -0,0 +1,286 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+"""
+Definition of high-level RethinkDB operations used by the RethinkDB check.
+
+Python ReQL reference documentation: https://rethinkdb.com/api/python/
+"""
+
+from typing import Any, Iterator, List, Mapping, Tuple
+
+import rethinkdb
+
+from .types import (
+    ClusterStats,
+    ConfigSummary,
+    ConnectionServer,
+    CurrentIssuesSummary,
+    Job,
+    JoinRow,
+    ReplicaStats,
+    Server,
+    ServerStats,
+    ServerStatus,
+    ShardReplica,
+    Table,
+    TableStats,
+    TableStatus,
+)
+
+# The usual entrypoint for building ReQL queries.
+r = rethinkdb.r
+
+# All system tables are located in this database.
+# See: https://rethinkdb.com/docs/system-tables/
+system = r.db('rethinkdb')
```

Comment:
Do we want this at import time?

Reply:
Sure, shouldn't be a problem. This doesn't make any I/O call, it just pre-builds an ReQL AST object that references the `'rethinkdb'` database.